docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                |  nRF52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF52840 DK     |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+
+> :information_source: In case you want to use nRF52840 Dongle as an OpenThread Radio Co-Processor read [nRF52840 dongle support](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf528xx/nrf52840/README.md#nrf52840-dongle-support-pca10059) and adjust the steps below accordingly.
+
+OpenThread RCP firmware is required to allow a PC to communicate with Thread devices. Complete the following steps to build and flash the RCP firmware onto a nRF52840 DK:
+
+1. Download and install the
+[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+2. Clone OpenThread repository into the current directory:
+
+        $ git clone https://github.com/openthread/openthread.git
+        
+3. Install OpenThread dependencies:
+
+        $ cd openthread
+        $ ./script/bootstrap
+4. Set up build environment:
+
+        $ ./bootstrap
+5. Build OpenThread for the nRF52840 DK:
+
+        $ make -f examples/Makefile-nrf52840
+
+6. Convert the resulting RCP image to HEX format:
+
+        $ arm-none-eabi-objcopy -O ihex output/nrf52840/bin/ot-rcp output/nrf52840/bin/ot-rcp.hex
+7. Flash the RCP firmware:
+
+        $ nrfjprog --chiperase --program output/nrf52840/bin/ot-rcp.hex --reset
+8. Disable Mass Storage feature on the device so that it doesn't [interfere](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf528xx/nrf52840/README.md#mass-storage-device-known-issue) with core RCP functionalities. Note that power-cycle of the device is required to apply the changes. The setting will remain valid even if you flash another firmware onto the device.
+
+        $ JLinkExe
+        J-Link>MSDDisable
+        Probe configured successfully.
+        J-Link>exit
+
+
+</details>
+
+## Configuring PC as a Thread border router
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+To make your PC work as a Thread border router, you need to complete the following tasks which can be further broken down into smaller steps:
+
+1. Form a Thread network using OpenThread Border Router Docker image and configure IPv6 packet routing to the network.
```

Comment:
Where do I get this image from?

Reply:
I want to avoid adding the same information several times. I added "which can be further broken down into smaller steps" piece to make a reader anticipate more precise explanation of the task. Should I really go into such details here?

Reply:
Maybe cross-link is enough? "further broken down into smaller steps" can suggest that I should either look further down or that some information is not provided, so it's not the best option.